manifests/e/Elgato/CameraHub/1.5.1.887/Elgato.CameraHub.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Elgato.CameraHub
+PackageVersion: 1.5.1.887
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: 1.3.0.595
```

Comment:
```suggestion
- DisplayVersion: 1.5.1.887
```

Reply:
I'm not sure why validation is erroring that the Apps & Features version is different, as it matches in my VM. 
![image](https://user-images.githubusercontent.com/114952565/208516846-9eae506b-1a44-43d3-a09b-2f7a147d965f.png)